src/contexts/playlists.context.js

```diff
@@ -0,0 +1,22 @@
+// import { createContext, useContext } from "react";
+
+// const PlaylistsContext = createContext()
+
+// export const PlaylistsProvider = ({ children }) => {
+//     const [playlistsState, playlistsDispatch] = useReducer(playlistsReducer, [])
+
+//     function playlistsReducer(state, action) {
+
+//         switch(action.type) {
+
+//         }
+//     }
+
+//     return (
+//         <PlaylistsContext.Provider>
+//             {children}
+//         </PlaylistsContext.Provider>
+//     )
+// }
+
+// export const usePlaylists = () => useContext(PlaylistsContext)
```

Comment:
If this is commented because you will use it in future then keep it else remove this file.

Reply:
thanks for pointing out. removed.